internal/worker/queue.go

```diff
@@ -123,6 +132,8 @@ func (q *queue) Push(ctx context.Context, job JobFunc) error {
 	}
 }
 
+// Pop returns (JobFunc, nil) if the channnel, which will be used for queuing job, contains JobFunc.
+// If pop returns error, Pos returns (nil, error)
```

Comment:
User do not know what is the difference between `pop` and `Pop`, Please describe more when will pop return error, or simply :
```suggestion
// If returns any error if it failed to pop from the job queue.
```

Reply:
```suggestion
// It returns (nil ,error) if it failed to pop from the job queue.
```
I think this sentence maybe correct 🤔 
@kevindiu